From: Michael Biebl <biebl@debian.org>
Date: Thu, 18 Jul 2013 20:36:39 +0200
Subject: Start logind on demand via libpam-systemd

Don't make pam_sm_open_session() a NOP if logind is not running. Trying
to access logind via D-Bus will start it on demand.
---
 src/login/pam-module.c | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

--- a/src/login/pam-module.c
+++ b/src/login/pam-module.c
@@ -222,10 +222,6 @@
 
         assert(handle);
 
-        /* Make this a NOP on non-logind systems */
-        if (!logind_running())
-                return PAM_SUCCESS;
-
         if (parse_argv(handle,
                        argc, argv,
                        &class_pam,
